File: src/BIDIM.WebApp/src/app/_viewModels/HouseholdViewModel.ts
export interface IHouseholdList {
  Id: string;
  FamilyName: string;
  MemberCount: number;
}

export interface IHouseholdViewModel {
  Id: number;
  FamilyName: string;
  CityMun: string;
  Brgy: string;
  Zone: string;
  Street: string;
  Long: number;
  Lat: number;
}

export interface IHouseholdDropdownList {
  Id: number;
  FamilyName: string;
}

File: src/BIDIM.Common/Models/HouseholdViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BIDIM.Common.Models
{
    public class HouseholdViewModel
    {
        public int Id { get; set; }

        public string FamilyName { get; set; }

        public string CityMun { get; set; }

        public string Brgy { get; set; }

        public string Zone { get; set; }

        public string Street { get; set; }

        public double Long { get; set; }

        public double Lat { get; set; }
    }
}
